packages/hosted-form-v2/src/iframe-content/hosted-input-events.ts

```diff
@@ -142,3 +146,16 @@ export interface HostedInputSubmitErrorEvent {
         response?: Response<PaymentErrorResponseBody>;
     };
 }
+
+export interface HostedInputStoredCardSucceededEvent {
+    type: HostedInputEventType.StoredCardSucceeded;
+}
+
+export interface HostedInputStoredCardErrorEvent {
+    type: HostedInputEventType.StoredCardFailed;
+    payload?: {
```

Comment:
Will there ever be a case that payload is not present?

Reply:
Saw this https://github.com/bigcommerce/checkout-sdk-js/pull/2767/files#diff-7bea3ab0d43b6b804c7f55d7e6f0d3bd485fde61e420afbd8265aa6942554770R58 so makes sense, but shall we return a payload there?

Reply:
Yes, we have similar comment in the initial PR. It gonna be done in separate ticket.
We need to improve error handling in general for the Stored cards functionality.

Reply:
We need to make strict types for the possible endpoint response errors.